### PR TITLE
unnaturalscrollwheels: init at 1.3.0

### PR DESCRIPTION
**Pull request by emilytrau**
###### Description of changes

Invert scroll direction for physical scroll wheels https://github.com/ther0n/UnnaturalScrollWheels

###### Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [x] aarch64-darwin
- [x] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Review by Enzime — APPROVED** [review]
LGTM, runs on `aarch64-darwin`

File: pkgs/tools/inputmethods/unnaturalscrollwheels/default.nix [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    mkdir -p $out/Applications
+    cp -r *.app $out/Applications
+
+    runHook postInstall
+  '';
+
+  meta = with lib; {
+    description = "Invert scroll direction for physical scroll wheels";
+    homepage = "https://github.com/ther0n/UnnaturalScrollWheels";
+    license = licenses.gpl3Plus;
```
Comment (Enzime) [inline pkgs/tools/inputmethods/unnaturalscrollwheels/default.nix:32]: Looks like the licence is `gpl3Only` as there are no licence notices stating

Comment (emilytrau) [reply pkgs/tools/inputmethods/unnaturalscrollwheels/default.nix:32]: I'm going off this line as the source https://github.com/ther0n/UnnaturalScrollWheels/blob/c0a1e2f52e12cb6b961aa888447acf4b827b6d12/LICENSE#L640

Comment (Enzime) [reply pkgs/tools/inputmethods/unnaturalscrollwheels/default.nix:32]: That's just the default text of the licence, but as the author didn't add licence notices to their files, I think GPL3-only is a safer assumption

**Review by drupol — APPROVED** [review]

**Pull request merged**